### PR TITLE
Fix errors arising from strict .mjs handling

### PR DESCRIPTION
**Pull request by developit**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -49,7 +49,38 @@ module.exports = function (_, env) {
       }
     },
     module: {
+      // Disable the default JavaScript handling:
+      defaultRules: [],
       rules: [
+        {
+          oneOf: [
```
Comment (tomayac) [inline webpack.config.js]: @developit Love your spelling of one-off here. Making clear it's really just a one-of.

Comment (developit) [reply webpack.config.js]: webpack gets the credit for this one, I'm just writing to their API :P

**Comment by developit** [comment]
A note: we can just remove this if preact ships its `module` export with the `.esm` extension. I've opened a PR there: developit/preact#1165

**Comment by mathiasbynens** [comment]
> A note: we can just remove this if preact ships its module export with the `.esm` extension.

Did you mean `.mjs`?

**Comment by developit** [comment]
yes. haha.

**Pull request merged**